Makefile

```diff
@@ -159,6 +159,12 @@ operator_whitelist_devnet:
 	@echo "Operator address: $(OPERATOR_ADDRESS)"
 	RPC_URL="http://localhost:8545" PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" OUTPUT_PATH=./script/output/devnet/alignedlayer_deployment_output.json ./contracts/scripts/whitelist_operator.sh $(OPERATOR_ADDRESS)
 
+operator_remove_devnet:
+	@echo "Whitelisting operator"
```

Comment:
I think this comment is incorrect, should be something similar to `Removing operator`